### PR TITLE
Refactored README

### PR DESCRIPTION
**Pull request by oskopek**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,66 +8,117 @@ A template for ML projects striking a balance between reproducibility and rapid
...
-To run Tensorboard in the notebook server, go to the specified log directory, and select New > Tensorboard. A pop-up window will open (make sure to enable those in your browser).
+The following sections describe the ways you can work with this template.
 
-Hint: Only do this once, in the `data_out/` folder.
+=== Jupyter notebooks
+
+Work on Colab, save notebooks in the `notebooks/` directory.
+These can also be worked on locally using Jupyter:
+
+* Run `jupyter notebook` in the project root directory.
```
Comment (jendelel) [inline README.adoc]: Run notebook **or** lab.

File: README.adoc [hunk 2]
```diff
@@ -8,66 +8,117 @@ A template for ML projects striking a balance between reproducibility and rapid
...
+
+To run Tensorboard locally alongside these, just run it with `--logdir data_out/`.
+
+==== Tensorboard
+
+To run Tensorboard in the notebook server directly, go to the specified log directory,
+and select New > Tensorboard.
+A pop-up window will open (make sure to enable those in your browser).
+
+*Hint:* Only do this once, in the `data_out/` folder.
```
Comment (jendelel) [inline README.adoc]: s/Hint/Note/

File: README.adoc (hunk 1 above)
Comment (oskopek) [reply README.adoc]: This is already done in #13

File: README.adoc (hunk 2 above)
Comment (oskopek) [reply README.adoc]: Done

**Pull request merged**